e2e/README.md

```diff
@@ -56,10 +56,10 @@ $ make e2e \
     SKIP_TEARDOWN=1
 ...
     To set kubecontext run:
-      export KUBECONFIG=/tmp/kots-e2e3629427925/.kubeconfig
+      export KUBECONFIG="$(k3d kubeconfig merge kots-e2e3629427925)"
```

Comment:
Haven't tested this yet but my understanding is that `k3d kubeconfig merge` will write a file, don't we want to use `k3d kubeconfig` get here instead?

Reply:
Can you set the env var to a string and not a filepath?